cli/commands.go

```diff
@@ -71,7 +71,7 @@ func Commands(metaPtr *meta.Meta) map[string]cli.CommandFactory {
 				CredentialBackends: map[string]logical.Factory{
 					"approle":  credAppRole.Factory,
 					"cert":     credCert.Factory,
-					"aws-ec2":  credAwsEc2.Factory,
```

Comment:
That's fair. I guess that's a bit more complicated but also a bit more explicit.